agimus_controller/agimus_controller/factory/robot_model.py

```diff
@@ -250,6 +285,15 @@ def _update_collision_model_to_self_collision(self) -> None:
             str(self._params.srdf.absolute()),
         )
 
+    def _add_collision_pairs(self) -> None:
```

Comment:
There should be a check with a proper error logging that lets user know if the collision pair exists in the geometry model or not